ci/jenkins/pipelines/prs/skuba-test.Jenkinsfile

```diff
@@ -181,6 +181,20 @@ pipeline {
             }
         } } }
 
+        stage('Addon update Check') { steps { script {
+            pr_context = 'jenkins/skuba-validate-addon-update'
+            sh(script: "${PR_MANAGER} update-pr-status ${pr_context} 'pending'", label: "Sending pending status")
+
+            try{
+                sh(script: "${PR_MANAGER} check-pr --manifest-check")
+                echo "No Addon issues detected"
+                sh(script: "${PR_MANAGER} update-pr-status ${pr_context} 'success'", label: "Sending success status")
+            } catch(err) {
+                echo "Unhandled error:\n${err}"
+                sh(script: "${PR_MANAGER} update-pr-status ${pr_context} 'failure'", label: "Sending failed status")
```

Comment:
need to raise the error here again, otherwise the pipeline will continue and we want it to fail no?

`error("Unhandled error:\n${err}")`